src/crimson/common/exception.h

```diff
@@ -18,6 +18,19 @@ class system_shutdown_exception final : public std::exception{
   }
 };
 
+class actingset_changed final : public std::exception {
+public:
+  actingset_changed(bool sp) : still_primary(sp) {}
+  const char* what() const noexcept final {
+    return "acting set changed";
+  }
+  bool is_primary() const {
+    return still_primary;
+  }
+private:
+  const bool still_primary = false;
```

Comment:
i'd suggest remove `= false`. as `still_primary` is always set in the constructor.